metis-media-service/src/main/java/eu/europeana/metis/mediaprocessing/extraction/ThumbnailGenerator.java

```diff
@@ -310,6 +314,40 @@ private ImageMetadata generateThumbnailsInternal(List<ThumbnailWithSize> thumbna
     return result;
   }
 
+  private String executeImageMagick(List<String> command) throws MediaExtractionException {
+    Path tempDir = createMagickTempDirectory();
+    try {
+      return commandExecutor.execute(command, singletonMap(MAGICK_TEMPORARY_PATH, tempDir.toString()), false,
```

Comment:
Note to investigate: Just to be sure. Is this temporary directory being created in ```/var/tmp/...``` or ```/tmp/...```? or do we need to checkout the property ```java.io.tmpdir``` of the system?

Reply:
Also check sonar security HOTSPOT, and remove code smells.

Reply:
> Note to investigate: Just to be sure. Is this temporary directory being created in `/var/tmp/...` or `/tmp/...`? or do we need to checkout the property `java.io.tmpdir` of the system?

java.io.tmpdir is set to special temp directory of storm worker